inference-contract/docs/notebooks/templates/token.py

```diff
@@ -0,0 +1,182 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.16.1
+#   kernelspec:
+#     display_name: Python 3 (ipykernel)
+#     language: python
+#     name: python3
+# ---
+
+
+# %% [markdown]
+# # Notebook to Invoke Token Operations
+
+# %% [markdown]
+# ## Configure Token Information
+#
+# This section enables customization of the token. Edit the variables in the section below as necessary.
+# * token_owner : the identity of the token owner
+# * token_class : the name of the class of tokens that was minted
+# * token_name : the name of the specific token that was minted
+# * token_path : the full context path to the token
+# * context_file : the name of the context file where token information is located
+#
+# Note that the notebook assumes that there is a key file for the identity of the form
+#
+# ```bash
+# ${keys}/${identity}_private.pem
+# ```
+#
+
+# %% tags=["parameters"]
+token_owner = 'user1'
+token_class = 'mytoken'
+token_name = 'token_1'
+token_path = 'token.${token_class}.token_object.${token_name}'
+context_file = '${etc}/${token_class}_context.toml'
+service_host = 'localhost'
+instance_identifier = ''
+
+# %%
+import os
+import pdo.contracts.jupyter as pc_jupyter
+import IPython.display as ip_display
+
+pc_jupyter.load_ipython_extension(get_ipython())
+
+
+# %% [markdown]
+# ### Initialize the PDO Environment
+#
+# Initialize the PDO environment. This assumes that a functional PDO configuration
+# is in place and that the PDO virtual environment has been activated. In particular,
+# ensure that the groups file and eservice database have been configured correctly.
+This can be done most easily by running the following in a shell:
+#
+# `$PDO_HOME/bin/pdo-create-service-groups.psh --service_host <service_host>`
+#
```

Comment:
because you might have come through this factory first. and... #32 will change the way this is handled completely. suggest we not push for changes at this time. this code is one of the prime motivations for #32.